README.md

```diff
@@ -4,16 +4,16 @@ SonarLint IntelliJ Plugin
 [![Build Status](https://api.cirrus-ci.com/github/SonarSource/sonarlint-intellij.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/sonarlint-intellij)
 [![Quality Gate](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij&metric=alert_status)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij)
 
-SonarLint is an IDE extension that helps you detect and fix quality issues as you write code.
+SonarLint is an IDE extension that helps you detect and fix quality issues as you write [Clean Code](https://www.sonarsource.com/solutions/clean-code/).
 Like a spell checker, SonarLint squiggles flaws so they can be fixed before committing code.
 
 Useful links
 ------------
 
 - [Documentation](https://docs.sonarsource.com/sonarlint/intellij/)
-  - A full list of supported programming languages and links to the static code analysis rules associated with each language is available on the [Rules page](https://docs.sonarsource.com/sonarlint/intellij/using-sonarlint/rules/).
+  - A full list of supported programming languages and links to the static code analysis rules associated with each language are available on the [Rules page](https://docs.sonarsource.com/sonarlint/intellij/using-sonarlint/rules/).
 - [Community](https://community.sonarsource.com/c/help/sl)
-  - Report an issue, ask for some help or suggest new features
+  - Report an issue, ask for some help, or suggest new features
```

Comment:
The sentence should have a `.` at the end.
Otherwise, the changes LGTM!

Reply:
Thanks, @kevin-hinz , I've added the period.